app/src/ble.c

```diff
@@ -65,13 +65,15 @@ static uint8_t active_profile;
 
 #define DEVICE_NAME CONFIG_BT_DEVICE_NAME
 #define DEVICE_NAME_LEN (sizeof(DEVICE_NAME) - 1)
+#define DEVICE_APPEARANCE                                                                          \
+    (uint8_t) CONFIG_BT_DEVICE_APPEARANCE, ((uint16_t)CONFIG_BT_DEVICE_APPEARANCE) >> 8
```

Comment:
Should probably be closer to:

```suggestion
    (uint8_t) CONFIG_BT_DEVICE_APPEARANCE, (uint8_t) (CONFIG_BT_DEVICE_APPEARANCE >> 8)
```